examples/safer_cluster/variables.tf

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  type        = string
+  description = "The project ID to host the cluster in"
+}
+
+variable "cluster_name_suffix" {
+  type        = string
+  description = "A suffix to append to the default cluster name"
+  default     = ""
+}
+
+variable "region" {
+  type        = string
+  description = "The region to host the cluster in"
+  default     = "us-central1"
+}
+
+variable "network" {
```

Comment:
I recommend removing most of these variables and replacing them with hard-coded values in the example. Additionally, any names should include a random component from a resource like `random_pet_name`.

Reply:
Done